platform/darwin/src/MGLStyleLayerManager.h

```diff
@@ -20,13 +20,27 @@ class LayerManagerDarwin : public LayerManager {
 
 private:
     LayerManagerDarwin();
+    /**
+     * @brief Enables a layer type for both JSON style and runtime API.
+     */
     void addLayerType(std::unique_ptr<LayerPeerFactory>);
+    /**
+     * @brief Enables a layer type for JSON style only.
+     *
+     * We might not want to expose runtime API for some layer types
+     * in order to save binary size - JNI glue code for these layer types
```

Comment:
Referring to JNI in Darwin code is confusing - is this a copy/paste?

Reply:
oops, thanks for the catch!